iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/CopyTreeImplTest.java

```diff
@@ -111,30 +111,14 @@ void copyInstructionWithOption() {
     ParamTree option = tree.options().get(0);
     assertThat(option.getKind()).isEqualTo(DockerTree.Kind.PARAM);
     assertThat(option.name()).isEqualTo("chown");
-    assertThat(option.value().value()).isEqualTo("55:mygroup");
+    assertThat(option.valuesAsString()).isEqualTo("55:mygroup");
 
     assertThat(tree.srcs()).hasSize(1);
     assertThat(tree.srcs().get(0).value()).isEqualTo("files*");
     assertThat(tree.dest().value()).isEqualTo("/somedir/");
   }
 
   @Test
-  void copyInstructionWithOptionWithoutValue() {
-    // This is not valid Docker syntax but should be parsed anyway
-    CopyTree tree = parse("COPY --option= src dest", DockerLexicalGrammar.COPY);
-    assertThat(tree.options()).hasSize(1);
-    assertTextRange(tree.textRange()).hasRange(1, 0, 1, 23);
-
-    ParamTree option = tree.options().get(0);
-    assertThat(option.getKind()).isEqualTo(DockerTree.Kind.PARAM);
-    assertThat(option.name()).isEqualTo("option");
-    assertThat(option.value()).isNull();
-
-    assertThat(tree.srcs()).hasSize(1);
-    assertThat(tree.srcs().get(0).value()).isEqualTo("src");
-    assertThat(tree.dest().value()).isEqualTo("dest");
-  }
-
   void copyInstructionHeredocForm() {
```

Comment:
Not a valid syntax as mentioned in the comment, we do not parse this anymore.
Also added @Test annotation to the test method `copyInstructionHeredocForm`.